src/pages/resultsView/cnSegments/CNSegments.tsx

```diff
@@ -143,7 +144,7 @@ export default class CNSegments extends React.Component<{ store: ResultsViewPage
                         ))
                     }
                 </Nav>
-                {!this.isLoading && (
+                <div style={this.isHidden ? {opacity: 0.1} : undefined}>
```

Comment:
hiding the actual component causes problems with rendering, so changing the opacity instead